### ISSUE TITLE
[BUG] Unable to select all albums when there is a large album list

### ISSUE THREAD

**Issue by LesKent** [giejay/Immich-Android-TV#25]
**Describe the bug**
I have large photo library with 2,839 albums. On the screensaver setup, the user interface for selecting albums to show, as I scroll down a pretty long ways it starts to get choppy/slow and if you keep scrolling downward it gets progressively slower until it hangs up for about 5 seconds. If I keep scroll down it will get to a point where it takes many seconds to respond to any more down selections. It then hangs for a while. After about 10 seconds it appears to crash and go back to the TV's home screen. I'm not sure how far it gets before the crash, a pretty long way, but I can't get to the end of the full album list.
 
**To Reproduce**
1. Have an Immich library with 2.8K+ albums. 
2. Select settings
3. Select Screensaver option
4. Select "Set albums to show in screensaver"
5. Keep scrolling downward until it gets choppy, hands and eventually crashes 

**Expected behavior**
Should be able to choose albums from the entire list

**My setup (please complete the following information):**
 - Android TV OS version: 11 (no updates available)
 - Device: TCL 65P725 TV
 - Immich app version: v1.101.0

**Comment by giejay** [comment]
I think a better solution would be to add a search option instead of needing to scroll through 1000s of albums.

**Comment by LesKent** [comment]
> I think a better solution would be to add a search option instead of needing to scroll through 1000s of albums.

Yes that makes sense to have a search function although the Google TV soft keyboard interface is pretty clunky for most things like this. I've not tried to connect a wireless USB keyboard to this TV yet which would make such an interface a lot more efficient. 

Another possible idea would be if the server side supported functionality for the screen saver and/or album viewing, such that you could use use a full web UI for such configuration and to minimize the TV interface.
